frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/PipelineRunCustomDetails.tsx

```diff
@@ -59,6 +65,37 @@ const PipelineRunCustomDetails: React.FC<PipelineRunCustomDetailsProps> = ({ pip
           <PipelineResourceRef {...convertBackingPipelineToPipelineResourceRefProps(pipelineRun)} />
         </dd>
       </dl>
+      {buildImage && (
+        <dl>
+          <dt>{t('pipelines-plugin~Download SBOM')}</dt>
+          <dd>
+            <ClipboardCopy
+              isReadOnly
+              hoverTip={t('pipelines-plugin~Copy')}
+              clickTip={t('pipelines-plugin~Copied')}
+            >
+              {`cosign download sbom ${buildImage}`}
+            </ClipboardCopy>
+            <ExternalLink href="https://docs.sigstore.dev/cosign/installation">
+              {t('pipelines-plugin~Install Cosign')}
+            </ExternalLink>
+          </dd>
+        </dl>
+      )}
+      {sbomTaskRun && (
+        <dl>
```

Comment:
There is no need to start new lists (`dl`) for each definition (`dt+dd`).

Can you clean this up at least for your definitions, and have one list for the Pipeline definition, SBOM, View SBOM, and the following three definitions (Start time, Completion time and Duration)?